.gitignore

```diff
@@ -29,3 +29,6 @@ __pycache__
 
 # Test results from 'make test'
 test-output.json
+
+tools/yamlfmt
```

Comment:
could you please add a comment line for future readers - what activity generates these files?